desktop/Makefile

```
@@ -0,0 +1,18 @@
+all: native frontend rust
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)